appcelerator/titanium_mobile
# TIMOB-7872: Mark all Android module APIs and filter them in Javadoc

To test this:

```
$ cd android
$ ant build.titanium.javadoc
$ open ../dist/android/javadoc/index.html
```

Verify that the _only_ APIs that you see are from this list:
https://wiki.appcelerator.org/display/guides2/Android+Module+API
